tests/rt.conf

```diff
@@ -115,7 +111,14 @@ RUN     | fv3_ccpp_gfs_v16_debug
 RUN     | fv3_ccpp_gfs_v15p2_RRTMGP_debug                                                                                         |                                         | fv3 |
 RUN     | fv3_ccpp_gfs_v16_RRTMGP_debug                                                                                           |                                         | fv3 |
 
-COMPILE | SUITES=FV3_GSD_v0,FV3_GFS_v15_thompson,FV3_RRFS_v1beta 32BIT=Y DEBUG=Y                                                  |                                         | fv3 |
+COMPILE | SUITES=FV3_GFS_v15_thompson_mynn 32BIT=Y DEBUG=Y                                                                        |                                         | fv3 |
+RUN     | fv3_ccpp_regional_control_debug                                                                                         |                                         | fv3 |
+
+COMPILE | SUITES=FV3_GFS_2017,FV3_GFS_2017_stretched 32BIT=Y DEBUG=Y                                                              |                                         | fv3 |
+RUN     | fv3_ccpp_control_debug                                                                                                  |                                         | fv3 |
+RUN     | fv3_ccpp_stretched_nest_debug                                                                                           |                                         | fv3 |
+
+COMPILE | SUITES=FV3_GSD_v0,FV3_GFS_v16_thompson,FV3_RRFS_v1beta 32BIT=Y DEBUG=Y                                                  |                                         | fv3 |
```

Comment:
Can we combine this COMPILE step with the two previous? All three have "32BIT=Y and DEBUG=Y" with different ccpp suites. We should try to minimize the number of times we run the build.

Reply:
Will do, good suggestion.

Reply:
Looks good to me.
There's only one thing, in .gitmodulefiles:

  #url = https://github.com/NOAA-EMC/fv3atm
  #branch = develop
  url = https://github.com/XiaSun-NOAA/fv3atm
  branch = gfsv16thomp

Reply:
Thanks, Ratko. Yes, this is temporary so that reviewers can check out the entire set of PRs recursively and run the regression tests. Once the submodule fv3atm gets merged, I need to update the submodule pointer and `.gitmodules` to point back to the official fork and branch.